resources/sass/app.scss

```diff
@@ -152,3 +152,40 @@ body {
     bottom: 0;
     width: 100%;
 }
+
+@include media-breakpoint-up(xs) {
+    .td-max-width-0-xs {
+        max-width: 0;
+    }
+}
+@include media-breakpoint-up(sm) {
+    .td-max-width-0-sm {
+        max-width: 0;
+    }
+}
+@include media-breakpoint-up(md) {
+    .td-max-width-0-md {
+        max-width: 0;
+    }
+}
+@include media-breakpoint-up(lg) {
+    .td-max-width-0-lg {
+        max-width: 0;
+    }
+}
+@include media-breakpoint-up(xl) {
+    .td-max-width-0-xl {
+        max-width: 0;
+    }
+}
+
+
+.text-ellipsis {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.actionColumn{
+    width: calc(126px + 0.75rem);
```

Comment:
What magic number are this?

Reply:
Could we use a calculation depending on the button widths?

Reply:
126px is the width of the three action buttons, 0.75rem the inner padding of the table cell

Reply:
Could we put this in variables?